### PR TITLE
[2195] Updated submodule references

### PR DESCRIPTION
**Pull request by quater**
I missed out to update the versions with the previous PR.

### CODE REVIEW

File: tests/local_requirements.yml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ---
 
 - src: sansible.users_and_groups
-  version: v1.0
+  version: v1.2.1
```
Comment (lobsterdore) [inline tests/local_requirements.yml]: I think this should be v1.2

File: tests/local_requirements.yml [hunk 2]
```diff
@@ -1,7 +1,7 @@
...
 
 - src: sansible.java
-  version: v1.0
+  version: v1.0.3
```
Comment (lobsterdore) [inline tests/local_requirements.yml]: This should stay at v1.0

File: tests/local_requirements.yml (hunk 1 above)
Comment (quater) [inline tests/local_requirements.yml]: Changed accordingly

File: tests/local_requirements.yml (hunk 2 above)
Comment (quater) [inline tests/local_requirements.yml]: Changed accordingly

**Comment by lobsterdore** [comment]
LGTM :+1:

**Pull request merged**